htlcswitch/quiescer.go

```diff
@@ -0,0 +1,215 @@
+//nolint:unused
```

Comment:
I don't believe so because I think there are parts of the API that are still unused even after this PRs final commit. I include them for completeness though. The only other way around this is to carve it off into a separate subpackage and give it a public API.